apps/communications/contacts/test/marionette/import_sdcard_test.js

```diff
@@ -0,0 +1,60 @@
+'use strict';
+
+var Contacts = require('./lib/contacts');
+var assert = require('assert');
+
+marionette('Import from SDCARD', function() {
+  var profile = Contacts.config;
+  // Enable device storage
+  var prefs = ['device.storage.enabled',
```

Comment:
Well, er, there is no need to change the configuration. The configuration is already the same as here: https://github.com/arcturus/gaia/blob/bug-1219636/apps/communications/contacts/test/marionette/lib/contacts.js#L20-L22